hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/MultiMapCodecTemplate.java

```diff
@@ -266,4 +266,15 @@
      */
     @Request(id = 21, retryable = false, response = ResponseMessageConst.BOOLEAN, partitionIdentifier = "key")
     Object removeEntry(String name, Data key, Data value, long threadId);
+
+    /**
+     * Removes all the entries with the given key. Returns void
```

Comment:
You may rather say that it does not return a response to the user. The protocol includes all client languages and void may not be very meaningful. You can also just not put this sentence since the response = ResponseMessageConst.VOID actually means the same thing.

Reply:
@ihsandemir thanks for the comments, I removed the part where it says `Returns void`.